config/locales/en.yml

```diff
@@ -94,7 +94,7 @@ en:
         favicon: Favicon
         favicon_info_text: "Favicon will replace the default Sharetribe logo. The dimensions should be 32x32 pixels and the uploaded image will be resized to these dimensions."
         community_custom_color1: "Main color"
-        community_custom_color2: "Second color"
+        community_custom_color2: "\"Post a new listing\" button color"
```

Comment:
Could make sense, no strong opinion about the use of italic, I rely on others "design" opinions.

If this is chosen, probably good to use the existing sentence https://webtranslateit.com/en/projects/2056-Sharetribe/locales/en..fr/strings/3354398 for the Post a new listing label.